05_syntax_lab/01.py

```diff
@@ -1,5 +1,18 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+# get 10 numbers and return the largest number
+
+print "please enter 10 numbers"
+
+x1 = int(raw_input ())
+x2 = int(raw_input ())
+x3 = int(raw_input ())
+x4 = int(raw_input ())
+x5 = int(raw_input ())
+x6 = int(raw_input ())
+x7 = int(raw_input ())
+x8 = int(raw_input ())
+x9 = int(raw_input ())
+x10 = int(raw_input ())
+
+
+print "the largest number is: ",  max (x1,x10)
+

```

Comment:
Welcome to the exercises!

While the above solution can work, we really don't want to repeat our code 10 times. What if tomorrow, you'll have to solve it for 100 or a 1000?
Please submit a solution that mentions ```raw_input()``` only once but calls it in a loop 10 times.